### DISCUSSION TITLE
Auto3DSeg supported modalities

### DISCUSSION THREAD

**Discussion by che85** [Project-MONAI/MONAI#7501] [Q&A] [answered]
Hello, 

is there any specific reason, why Auto3DSeg only supports CT and MRI modalities?

Thank you, 
Christian

**Comment by dongyang0122** [comment, marked as answer]
Hi @che85 , most of the public datasets are based on CT images and MRIs as we observed in many MICCAI challenges. The modality determines some transformations and hyper-parameters in each algorithm. But if you want to try Auto3DSeg on other modes, such as ultrasound, you can set the modality to MRI as a quick workaround. For now, we'll see if we get enough feedback from users to develop for additional modalities.
